### ISSUE TITLE
Design newsletter signup box

### ISSUE THREAD

**Issue by hagenburger** [eurucamp/livingstyleguide-eurucamp#9]

**Comment by bascht** [comment]
Still valid? If so: What is the specific task here? Customize the [mailchimp form](http://eurucamp.us1.list-manage1.com/subscribe?u=c36464acd69c447ab9037be73&id=d27edde894)?

**Comment by hagenburger** [comment]
Yes, that’s a bit ugly and would look better inline in our site. But I wouldn’t give a high priority to it.

**Comment by bascht** [comment]
Alright - just checking. :)
